angular/src/lib/list-separator/list-separator.component.ts

```diff
@@ -0,0 +1,44 @@
+import { Component, HostBinding, Input } from '@angular/core';
+
+@Component({
+  selector: 'cui-list-separator',
+  template: `
+    <span class="cui-list-separator__container">
+      <ng-content *ngIf="!text; else textTpl"></ng-content>
```

Comment:
This is ok, but I would prefer we spell variable names out and not abbreviate, especially when the are internal to the component and not used by the consumer. 
`textTemplate`

Reply:
updated